src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -335,7 +336,9 @@ private void processFiles(RootModule rootModule, final SeverityLevelCounter warn
         log("To locate the files took " + (endTime - startTime) + TIME_SUFFIX,
             Project.MSG_VERBOSE);
 
-        log("Running Checkstyle " + checkstyleVersion + " on " + files.size()
+        log("Running Checkstyle "
+                + Objects.toString(checkstyleVersion, "")
+                + " on " + files.size()
```

Comment:
Does it matter that we are printing double spaces for no version?
```
Running Checkstyle  on X files (2 spaces)
Running Checkstyle on X files (1 space)
```

Reply:
Doesn't matter. Any extra if will force us to deal with coverage and pitest